### ISSUE TITLE
Custom Fonts Need Example

### ISSUE THREAD

**Issue by justin808** [shakacode/react-webpack-rails-tutorial#8]
When using custom fonts, there's an issue in that the sass helper font-url does not work with lib-sass.

The only way I've found around this is to have different files loaded for the boostrap-customization for each of the Rails and Webpack parts, and have these call bootstrap-customization-common.scss
# Change Webpack File

This file needs to refer to a file that is webpack specific, like "boostrap-variables-customization-webpack"
and have that file `@import "bootstrap-variables-customization";`

https://github.com/justin808/react-webpack-rails-tutorial/blob/master/webpack/bootstrap-sass.config.js

```
module.exports = {
  bootstrapCustomizations: "./assets/stylesheets/_bootstrap-variables-customization.scss",
```
# Change Rails File

This line should refer to something like "boostrap-variables-customization-rails" and have that file `@import "bootstrap-variables-customization";`

https://github.com/justin808/react-webpack-rails-tutorial/blob/master/app/assets/stylesheets/_bootstrap-custom.scss#L2

```
@import "bootstrap-variables-customization";
```
# Fix up some imports from the Rails side

These lines should get referenced by 
https://github.com/justin808/react-webpack-rails-tutorial/blob/master/app/assets/stylesheets/application.css.scss#L1

By a line at the end of 
https://github.com/justin808/react-webpack-rails-tutorial/blob/master/app/assets/stylesheets/_bootstrap-custom.scss#L53

And this file needs to exist, that's referred to in the https://github.com/justin808/react-webpack-rails-tutorial/blob/master/webpack/bootstrap-sass.config.js#L6:

```
 mainSass: "./_main.scss",
```

Basically, that file, _main.scss will have access to override any bootstrap styles and will be able to use any bootstrap variables.

The key thing to keep in mind is that the custom variables must be declared right after the bootstrap default variables are loaded and **before** the bootstrap styles are loaded, and then the app specific custom CSS should run.

I'm also guessing that custom font declarations should go in the place that the custom variables are defined, as the custom variables could indicate fonts that are custom.
